calypso/csadmin/commands.go

```diff
@@ -117,6 +117,10 @@ var cmds = cli.Commands{
 								EnvVar: "BC",
 								Usage:  "the ByzCoin config to use (required)",
 							},
+							cli.StringFlag{
+								Name:  "roster",
+								Usage: "change the roster for the calypso",
```

Comment:
for me something like "the path of a roster file to be used as argument for the spawn. If not provided the config roster is used (optional)" would be more understandable.